api/swagger.yaml

```diff
@@ -6858,6 +6858,42 @@ definitions:
               example: "24"
       FirewallBackend:
         $ref: "#/definitions/FirewallInfo"
+      DiscoveredDevices:
+        description: |
+          List of devices discovered by device drivers.
+
+          Each device includes information about its source driver, kind, name,
+          and additional driver-specific attributes.
+        type: "array"
+        items:
+          type: "object"
```

Comment:
Possibly would be worth to define the item in the array as a definition, instead of the `DiscoveredDevices`, so


```yaml
type: "array"
    items: $ref:...
```

Reply:
Done